src/main/java/me/mrCookieSlime/Slimefun/Lists/SlimefunItems.java

```diff
@@ -267,6 +266,35 @@ private SlimefunItems() {}
 
     public static final SlimefunItemStack BOOTS_OF_THE_STOMPER = new SlimefunItemStack("BOOTS_OF_THE_STOMPER", Material.LEATHER_BOOTS, Color.AQUA, "&bBoots of the Stomper", "", "&9All Fall Damage you receive", "&9will be applied to nearby Mobs/Players", "", "&9+ No Fall Damage");
 
+    public static final SlimefunItemStack BEE_HELMET;
+    public static final SlimefunItemStack BEE_WINGS;
+    public static final SlimefunItemStack BEE_LEGGINGS;
+    public static final SlimefunItemStack BEE_BOOTS;
+
+static {
+
+    if (SlimefunPlugin.getMinecraftVersion().isAtLeast(MinecraftVersion.MINECRAFT_1_15)) {
```

Comment:
These items actually don't use any 1.15 Materials, so there isn't really much of a point in initializing them conditionally here. It's fine for these items to exist on 1.13/1.14 since you already handle that in SlimefunItemSetup

Reply:
Good point, I'll just go ahead and replace the recipe with something else for the versions below 1.15.X

Reply:
nononononononoononnoonoononono.

The way you handled it in SlimefunItemSetup was totally correct.
It's just redundant at this spot since you have already done it in SlimefunItemSetup.
Do **not** introduce alternate recipes because that will just be a total mess and frankly, having bee armor prior to 1.15 doesn't really make much sense anyway.

Reply:
Ah, I understand. Its indeed a bit wierd to have bee themed armor in a version without bees.